src/lib/logging.ts

```diff
@@ -7,6 +7,7 @@ import {
   EmbedBuilder,
   time,
   TimestampStyles,
+  ThreadChannel,
```

Comment:
Since it's only used as a type in this file, let's change this to a type import.
```suggestion
  type ThreadChannel,
```